File: backend/Metropolitan/Common/model/formaPago.cs
using System;

namespace Common.model
{
    public class formaPago
    {
        public int id;
        public string nombre;
        public string moneda; // 'USD' | 'BS' | 'AMBOS'
        public string areasAplicables; // e.g. 'TRAVELACE,PAQUETES,CARGA'
        public bool requiereCuentaBancaria;
        public bool activo;
        public int orden;
        public int createBy;
        public DateTime createDate;
        public int modify;
        public DateTime modifyDate;
    }
}

File: frontend/metropolitan/src/app/shared/model/forma-pago.ts
export interface FormaPago {
  id?: number;
  nombre: string;
  moneda: string; // 'USD' | 'BS' | 'AMBOS'
  areasAplicables?: string; // e.g. 'TRAVELACE,PAQUETES,CARGA'
  requiereCuentaBancaria: boolean;
  activo: boolean;
  orden?: number;
  createBy?: number;
  modify?: number;
  createDate?: string;
  modifyDate?: string;
}
